### PR TITLE
Remove unnecessary files

### PR DESCRIPTION
**Pull request by hayuna**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,11 +1,10 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import './index.css';
-import App from './App';
-import * as serviceWorker from './serviceWorker';
-import 'semantic-ui-less/semantic.less'
+import React from "react";
```
Comment (AnBera) [inline src/index.js]: Can we stick to the single quote for string. Also can we do a basic setup for linting.
I would recomment to follow airbnb rules.. The .eslintrc file look something like this

```
{
  "extends": "airbnb",
  "parserOptions": {
    "sourceType": "module",
    "ecmaFeatures": {
      "jsx": true
    }
  },
  "env": {
    "browser": true,
    "commonjs": true,
    "es6": true,
    "jquery": true,
    "jest": true
  },
  "rules": {
    "react/jsx-filename-extension": 0,
    "jsx-a11y/label-has-for": 0,
    "jsx-a11y/label-has-associated-control": 0
  }
}
```

File: package.json [hunk 2]
```diff
@@ -32,11 +32,5 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
-  },
-  "devDependencies": {
-    "@craco/craco": "^5.6.3",
-    "@semantic-ui-react/craco-less": "^1.2.1",
-    "semantic-ui-less": "^2.4.1",
-    "semantic-ui-react": "^0.88.2"
```
Comment (AnBera) [inline package.json:40]: Looks good. If we need sematic Ui react we will add later

**Review by AnBera — CHANGES_REQUESTED** [review]
a basic liting setup would be helpful.

**Comment by hayuna** [comment]
package-lock.json / yarn.lock should be added to repository only if we are using specific version of packages.

**Review by AnBera — APPROVED** [review]

**Pull request merged**